tools/msvs/msi/product.wxs

```diff
@@ -23,8 +23,8 @@
              Compressed="yes"
              InstallScope="perMachine"/>
 
-    <Condition Message="This application is only supported on Windows 8.1, Windows Server 2012 R2, or higher.">
-      <![CDATA[Installed OR (VersionNT >= 603) OR (VersionNT >= 602 AND MsiNTProductType <> 1)]]>
+    <Condition Message="This application is only supported on Windows 7, Windows Server 2008 R2, or higher.">
+      <![CDATA[Installed OR (VersionNT >= 601)]]>
```

Comment:
Is this something that blocks installs or just a message that’s shown?

Reply:
I'm not sure about this myself either. It was part of the original commit that put the EOL exit in place. Not sure where it's used (or if at all) myself.

Reply:
It blocks: https://wixtoolset.org/documentation/manual/v3/howtos/redistributables_and_install_checks/block_install_on_os.html

I think we should not change the message, or if we do change the message it should not say Windows 7/Server 2008 are supported. Maybe something like "This application will only install on Windows 7, Windows Server 2008 R2, or higher but is only supported on Windows 8.1, Windows Server 2012 R2, or higher." (i.e. make a distinction on what is allowed vs what is actually supported).

Reply:
> It blocks: https://wixtoolset.org/documentation/manual/v3/howtos/redistributables_and_install_checks/block_install_on_os.html
> 
> I think we should not change the message, or if we do change the message it should not say Windows 7/Server 2008 are supported. Maybe something like "This application will only install on Windows 7, Windows Server 2008 R2, or higher but is only supported on Windows 8.1, Windows Server 2012 R2, or higher." (i.e. make a distinction on what is allowed vs what is actually supported).

Thank you for that insight, I have added a new commit that resolves this as I was unsure of what this was related to. :slightly_smiling_face: